modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/NavigationCardTag.java

```diff
@@ -121,7 +192,25 @@ protected int processStartTag() throws Exception {
 			jspWriter.write("aspect-ratio-item-center-middle ");
 			jspWriter.write("aspect-ratio-item-flush\">");
 
-			return EVAL_BODY_INCLUDE;
+			String icon = getIcon();
+
+			if (icon != null) {
```

Comment:
What if both `icon` and `imageSrc` are defined? Will both be rendered?

Reply:
If the card is big and both are set, yes. If the card is small, only `icon` will be rendered. In this we are following the condition logic from [Clay component](https://github.com/liferay/clay/blob/master/packages/clay-card/src/CardWithNavigation.tsx), the only difference being instead of `{children}` we have the hardcoded `<img>`.

I think this is fine, it's a minor edge case. Who knows, maybe there could even be an odd case where we would want icon superimposed on image, that does something different than navigation :man_shrugging:

Reply:
Taking a look to [Lexicon definition](https://liferay.design/lexicon/core-components/cards/cards-navigation/): 

>This card is simple containing just a minimalistic image or icon, title and possible short description.

It feels like only one of them should be rendered, it actually feels weird to me to render both. It's indeed an edge case (nobody should pass both attributes) but just in case I'd safe check it.